content/en/blog/_posts/2024-10-20-Unlocking-the-power-of-Kubernetes-for-AI-Workloads.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: blog
+title: "Unlocking the Power of Kubernetes for AI Workloads"
+date: 2024-10-20
+slug: kubernetes-for-ai-workloads
+author: >
+  Brandon Kang (Akamai Technologies)
```

Comment:
Please add a line `draft: true` here; that will make it easier to merge this and then sort out publication later.